sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/aio/_client.py

```diff
@@ -171,13 +176,13 @@ def key_id(self) -> str:
                 raise AzureError("This client doesn't have 'keys/encrypt' permission")
             result = local_key.encrypt(plaintext, algorithm=algorithm.value)
         else:
-            result = await self._client.encrypt(
+            operation = await self._client.encrypt(
                 self._key_id.vault_endpoint, self._key_id.name, self._key_id.version, algorithm, plaintext, **kwargs
-            ).result
+            )
+            result = operation.result
```

Comment:
Sanity checking that this didn't change the rtype as it was documented?  (Saw that this changed but that didn't)

Reply:
Correct. Trying to make this less confusing by making the unpacking of the service response more obvious and removing `result.result`.